### PR TITLE
Added java model for point

### PR DESCRIPTION
**Pull request by 47souls**

### CODE REVIEW

**Comment by KostyaTk** [comment]
Reviewed. But we will need model for Places too.

File: src/main/java/ua/softserve/rv_018/greentourism/model/Point.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "points")
```
Comment (Sabfir) [inline src/main/java/ua/softserve/rv_018/greentourism/model/Point.java:11]: I'd like you to rename it to the singular "point"
Check this link, pls.
http://stackoverflow.com/questions/338156/table-naming-dilemma-singular-vs-plural-names

File: src/main/java/ua/softserve/rv_018/greentourism/model/Point.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+	private Long id;
+	@Column(name = "langtitude")
+	private String langtitude;
+	@Column(name = "longtitude")
+	private String longtitude;
+
+	public Point() {
+	}
+
+	public Point(Long id, String langtitude, String longtitude) {
```
Comment (Sabfir) [inline src/main/java/ua/softserve/rv_018/greentourism/model/Point.java:25]: Pls, delete this constructor if you are not using it.
We discussed this on Friday

**Comment by Sabfir** [comment]
By points we mean Places and Points

**Comment by RoxVer** [comment]
I use this empty constructor by the way, so no need to delete it.

**Comment by Sabfir** [comment]
Empty constructor is used by Spring so it's obligatory to have it.
You are doing comments in closed issue, be attentive.

**Comment by RoxVer** [comment]
So which constructor do you want to delete?

I see that.

**Comment by Sabfir** [comment]
I added inline comment so that everyone could see what constructor I'm talking about.
Inline comment is the comment, that refers to the code line just above.

**Pull request closed without merging**